tv-shows/src/components/features/review/ReviewItem/ReviewItem.tsx

```diff
@@ -2,13 +2,28 @@ import { IReviewItem } from "@/typings/Review";
 import { Flex, IconButton, Image, Text } from "@chakra-ui/react";
 import { DeleteIcon } from '@chakra-ui/icons';
 import { Card, CardHeader, CardBody, CardFooter } from '@chakra-ui/react'
+import useSWRMutation from "swr/mutation";
+import { swrKeys } from "@/fetchers/swrKeys";
+import { authenticatedDeleter } from "@/fetchers/authenticatedMutators";
 
 interface IReviewItemProps {
     reviewItem: IReviewItem;
     onDeleteReview: (review : IReviewItem) => void;
 }
 
 export const ReviewItem = ({ reviewItem, onDeleteReview }: IReviewItemProps) => {
+    const { trigger } = useSWRMutation(swrKeys.review(reviewItem.id.toString()), authenticatedDeleter);
+
+    const onClickAction = async (reviewItem : IReviewItem) => {
+        await trigger(reviewItem.id);
+        onDeleteReview(reviewItem);
```

Comment:
Ovo koliko sam skužio samo updatea lokalni state.
To nam više ne treba jer unutar useSWRMutation možemo postaviti onSuccess prop kojem kažemo da mutira naš lokalni state.
Tako bi onda vjerojanto mogli maknuti i onaj glomazni useEffect iz ShowContainer komponente